modules/LemDCBM400600/main/lem_dcbm_400600_controller.cpp

```diff
@@ -51,6 +51,33 @@ std::string LemDCBM400600Controller::get_current_transaction() {
     }
 }
 
+void LemDCBM400600Controller::update_lem_status() {
+    // should call this after a communication error to figure out what has been happening
+    auto status_response = this->http_client->get("/v1/status");
```

Comment:
try to move hardcoded string literals like `/v1/status` into constant expressions (constants).